client/src/components/requests/RequestedList.tsx

```diff
@@ -0,0 +1,151 @@
+import React from "react";
+import {
+    Card,
+    Container,
+    Header,
+    Label,
+    Step
+} from "semantic-ui-react";
+import {RequestedItem} from "../../types/Hardware";
+import {useQuery} from "@apollo/react-hooks";
+import {DESK_REQUESTS} from "../util/graphql/Queries";
+import {Request} from "../../types/Request";
+
+interface RequestedListProps {
+    requestedItemsList: RequestedItem[],
+}
+
+function RequestedList({requestedItemsList}: RequestedListProps) {
+    const {loading, error, data} = useQuery(DESK_REQUESTS);
+    if (loading) return <p>Loading...</p>;
```

Comment:
If this is inside a Segment, you can just set the loading={true} property on that and Semantic will show a loading spinner.

If that's not an option, you can also show a centered loading spinner like this:
`        return <Loader active inline="centered" content="Please wait..."/>;`